packages/accounts-password/password_server.js

```diff
@@ -500,11 +500,16 @@ Accounts.setPassword = function (userId, newPlaintextPassword, options) {
 Meteor.methods({forgotPassword: function (options) {
   check(options, {email: String});
 
-  var user = Meteor.users.findOne({"emails.address": options.email});
+  var user = Accounts._findUserByQuery({email: options.email});
   if (!user)
     throw new Meteor.Error(403, "User not found");
 
-  Accounts.sendResetPasswordEmail(user._id, options.email);
+  const emails = _.pluck(user.emails || [], 'address');
+  const caseSensitiveEmail = _.find(emails, email => {
```

Comment:
My reasoning was that the ability to specify a case insensitive email is a convenience, but email addresses should still be treated as case sensitive. Only allowing this in `forgotPassword` and keeping `sendResetPasswordEmail` case sensitive seemed to make sense for consistency with `sendEnrollmentEmail` and `sendVerificationEmail`.

What use case do you have in mind for invoking `sendResetPasswordEmail` manually on the server?